connect/transforms/src/main/java/org/apache/kafka/connect/transforms/Cast.java

```diff
@@ -67,6 +67,7 @@ public abstract class Cast<R extends ConnectRecord<R>> implements Transformation
                     + "or value (<code>" + Value.class.getName() + "</code>).";
 
     public static final String SPEC_CONFIG = "spec";
+    public static final String REPLACE_NULL_WITH_DEFAULT = "replace.null.with.default";
```

Comment:
Config key constants are generally suffixed with `_CONFIG`
```suggestion
    public static final String REPLACE_NULL_WITH_DEFAULT_CONFIG = "replace.null.with.default";
```

Here and throughout.

Reply:
This was the original name but @mimaison asked to remove the suffix to be aligned with others fields. https://github.com/apache/kafka/pull/15756#discussion_r1662794000

Reply:
In some files we have the `_CONFIG` suffix, for example `Cast` but in some others like `HeadersFrom` we don't. When I asked you to make changes I meant so it's consistent in each file. Sorry if I wasn't clear enough.

Reply:
So you meant consistent per file not at whole. It seems so weird but ok I'll then adjust it to be per file consistent.

Reply:
@gharris1727 Changes done.

Reply:
Sorry I didn't look at the previous review comments to see that this was already discussed. I am fine with _CONFIG everywhere, or _CONFIG only in files that have _CONFIG for the other variables. Thank you for changing this to be more consistent!